server_config.go

```diff
@@ -67,7 +70,7 @@ func (c *PacketConnConfig) validate() error {
 // ListenerConfig is a single net.Listener to accept connections on. This will be used for TCP, TLS and DTLS listeners
 type ListenerConfig struct {
 	Listener net.Listener
-
+	Protocol string // TCP, or TLS
```

Comment:
Please indicate the expected string constants. E.g. "tcp", "tls" in the comment.